File: sdk-node/examples/features/unarchive.ts
/**
 * Reactivate a previously archived feature. Idempotent.
 *
 * Run:
 *   npx tsx examples/features/unarchive.ts
 */

import { ThreeCommon } from '@3common/sdk'

const client = new ThreeCommon({
  apiKey: '3co_your_api_key_here',
})

const feature = await client.features.unarchive('feat_replace_with_real_id')

console.log(`unarchived ${feature.id ?? '?'} — active=${String(feature.active ?? false)}`)

File: sdk-python/examples/features/unarchive.py
"""Reactivate a previously archived feature. Idempotent.

Run with: python examples/features/unarchive.py
"""

from __future__ import annotations

from threecommon import ThreeCommon


def main() -> None:
    with ThreeCommon(api_key="3co_your_api_key_here") as client:
        feature = client.features.unarchive("feat_replace_with_real_id")
        print(f"unarchived {feature.id} — active={feature.active}")


if __name__ == "__main__":
    main()
